.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*.{js,ts,yaml,json}]
```

Comment:
Do you also need to specify jsx, tsx?

Reply:
Added.

Was trying to limit the impact on other contributors, but this shouldn't really break anybody's workflow.